recipes-support/mstpd/mstpd_git.bb

```diff
@@ -5,14 +5,12 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=4325afd396febcb659c36b49533135d4"
 
 SRC_URI = "\
   git://github.com/mstpd/mstpd.git;protocol=https \
-  file://0001-libnetlink-introduce-rta_nest-and-u8-u16-u64-helpers.patch \
-  file://0002-libnetlink-Set-NLA_F_NESTED-in-rta_nest.patch \
```

Comment:
Why are you removing these lines? Please explain in your commit message.

Also please delete the files itself.